tests/html/click-tests.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+        <title>Gamepad Navigator Tabbing Tests</title>
+
+        <link rel="stylesheet" media="screen" href="../../node_modules/infusion/tests/lib/qunit/css/qunit.css" />
+        <link rel="stylesheet" href="../../node_modules/infusion/tests/lib/qunit/addons/composite/qunit-composite.css">
+
+        <script type="text/javascript" src="../../node_modules/infusion/dist/infusion-all.min.js"></script>
+        <script type="text/javascript" src="../../node_modules/ally.js/ally.min.js"></script>
+        <script type="text/javascript" src="../../node_modules/infusion/tests/lib/qunit/js/qunit.js"></script>
+        <script type="text/javascript" src="../../node_modules/infusion/tests/test-core/jqUnit/js/jqUnit.js"></script>
+
+        <script type="text/javascript" src="../../src/js/content_scripts/gamepad-navigator.js"></script>
+        <script type="text/javascript" src="../../src/js/content_scripts/input-mapper-utils.js"></script>
+        <script type="text/javascript" src="../../src/js/content_scripts/input-mapper.js"></script>
+
+        <script type="text/javascript" src="../js/gamepad-mock.js"></script>
+        <script type="text/javascript" src="../js/click/click-mock-utils.js"></script>
+        <script type="text/javascript" src="../js/click/click-basic-tests.js"></script>
+        <script type="text/javascript" src="../js/click/click-dropdown-tests.js"></script>
+    </head>
+    <body id="body">
+        <h1 id="qunit-header">Gamepad Navigator Click Tests</h1>
+        <h2 id="qunit-banner"></h2>
+        <div id="qunit-testrunner-toolbar"></div>
+        <h2 id="qunit-userAgent"></h2>
+        <ol id="qunit-tests"></ol>
+
+        <div class="test-elements">
```

Comment:
I was curious about this earlier, did you try putting your test fixtures in the hidden div that QUnit uses?  I sometimes have to avoid using that, just wanted to know if that's also the case here.

Reply:
I wasn't aware of the fact that QUnit has a variation in regard to the types of divs it uses, so I haven't tried that before. Does it have a special use case in the tests? I'll try to go through it as it's something I haven't heard or read before.

Reply:
It's not well documented in the QUnit docs, but there is [a brief bit on it in ours](https://docs.fluidproject.org/infusion/development/jqUnit.html#testing-in-the-browser).  The only difference is a) the markup is not displayed and b) the markup is reset between runs. Like I said, I was just curious, I'd be happy with horizontal rule between the QUnit markup and yours just to distinguish.

Reply:
Thank you for sharing this. I'll check it out.